locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "vibraphone": "वाइब्राफ़ोन",
+  "cello": "चेलो",
+  "double bass": "डबल बास",
+  "guitar": "गिटार",
+  "sitar": "सितार",
+  "harmoiunm": "harmoiunm",
+  "acoustic guitar": "एकॉस्टिक गिटार",
+  "flute": "बांसुरी",
+  "clarinet": "क्लैरिनेट",
+  "saxophone": "सैक्सोफोन",
+  "tuba": "ट्यूबा",
+  "trumpet": "ट्रंपेट",
+  "oboe": "ओबो",
+  "trombone": "ट्रॉम्बोन",
+  "electronic synth": "इलेक्ट्रॉनिक सिंथ",
+  "simple 1": "सिम्पल 1",
+  "simple 2": "सिम्पल 2",
+  "simple 3": "सिम्पल 3",
+  "simple 4": "सिम्पल 4",
+  "white noise": "व्हाइट नॉइज़",
+  "brown noise": "ब्राउन नॉइज़",
+  "pink noise": "पिंक नॉइज़",
+  "snare drum": "स्नेयर ड्रम",
+  "kick drum": "किक ड्रम",
+  "tom tom": "टॉम टॉम",
+  "floor tom": "फ्लोर टॉम",
+  "bass drum": "बेस ड्रम",
+  "cup drum": "कप ड्रम",
+  "darbuka drum": "दरबूका ड्रम",
+  "hi hat": "हाई हैट",
+  "ride bell": "राइड बेल",
+  "cow bell": "काउ बेल",
+  "japanese drum": "जापानी ड्रम",
+  "japanese bell": "जापानी बेल",
+  "triangle bell": "त्रिकोण बेल",
+  "finger cymbals": "फिंगर सिम्बल्स",
+  "chime": "चाइम",
+  "gong": "गॉंग",
+  "clang": "क्लैंग",
+  "crash": "क्रैश",
+  "bottle": "बॉटल",
+  "clap": "क्लैप",
+  "slap": "स्लैप",
+  "splash": "स्प्लैश",
+  "bubbles": "बबल्स",
+  "raindrop": "रेनड्रॉप",
+  "cat": "बिल्ली",
+  "cricket": "क्रिकेट",
+  "dog": "कुत्ता",
+  "banjo": "बैंजो",
+  "koto": "कोतो",
+  "dulcimer": "डल्सिमर",
+  "electric guitar": "इलेक्ट्रिक गिटार",
+  "bassoon": "बैसून",
+  "celeste": "सेलेस्ट",
+  "equal": "इक्वल",
+  "Pythagorean": "पिथागोरियन",
+  "just intonation": "जस्ट इंटोनेशन",
+  "Meantone": "मींटोन",
+  "major 7th": "मेजर 7वां",
+  "minor 7th": "माइनर 7वां",
+  "dominant 7th": "डॉमिनेंट 7वां",
+  "minor-major 7th": "माइनर-मेजर 7वां",
+  "fully-diminished 7th": "फुली-डिमिनिश 7वां",
+  "half-diminished 7th": "हाफ-डिमिनिश 7वां",
+  "Equal (12EDO)": "समान (12EDO)",
+  "Equal (5EDO)": "समान (5EDO)",
+  "Equal (7EDO)": "समान (7EDO)",
+  "Equal (19EDO)": "समान (19EDO)",
+  "Equal (31EDO)": "समान (31EDO)",
+  "5-limit Just Intonation": "5-सीमा सही स्वर ताल",
+  "Pythagorean (3-limit JI)": "पाइथागोरियन (3-सीमा JI)",
+  "current": "वर्तमान",
+  "next": "अगला",
+  "previous": "पिछला",
+  "harmonium": "harmonium",
+  "simple-2": "सिंपल-2",
+  "simple-3": "सिंपल-3",
+  "simple-4": "सिंपल-4",
+  "taiko": "ताइको",
+  "The Not block is the logical not operator.": "नॉट ब्लॉक तार्किक नॉट ऑपरेटर है।",
+  "not": "नहीं",
+  "The And block is the logical and operator.": "एंड ब्लॉक तार्किक एंड ऑपरेटर है।",
+  "and": "और",
+  "The Or block is the logical or operator.": "ओर ब्लॉक तार्किक ओर ऑपरेटर है।",
+  "or": "या",
+  "The XOR block is the logical XOR operator.": "XOR ब्लॉक तार्किक XOR ऑपरेटर है।",
+  "xor": "एक्सओआर",
+  "The Equal block returns True if the two numbers are equal.": "इक्वल ब्लॉक यदि दो नंबर समान हैं तो True लौटाता है।",
+  "The Not-equal-to block returns True if the two numbers are not equal to each other.": "नॉट-इक्वल-टू ब्लॉक तब ट्रू लौटाता है जब दो नंबर एक-दूसरे के बराबर न हों।",

```

What do you think about writing 'ट्रू' as 'सत्य'. I am fine with transliterating it like you did here but would users understand it better it if is सत्य ?

```suggestion
  "The Not-equal-to block returns True if the two numbers are not equal to each other.": "'नॉट-इक्वल-टू' ब्लॉक तब 'ट्रू' लौटाता है जब दो नंबर एक-दूसरे के बराबर न हों।",

```